src/Behat/Behat/Output/Node/EventListener/AST/ScenarioNodeListener.php

```diff
@@ -83,9 +83,9 @@ public function listenEvent(Formatter $formatter, Event $event, $eventName)
      * @param ScenarioLikeTested|AfterSetup $event
      * @param string                        $eventName
      */
-    private function printHeaderOnBeforeEvent(Formatter $formatter, ScenarioLikeTested $event, $eventName)
+    private function printHeaderOnBeforeEvent(Formatter $formatter, ScenarioLikeTested|AfterSetup $event, $eventName)
```

Comment:
The event can implement any of these interfaces

Reply:
no need to change this native type. The check in `listenEvent` ensures that we always get a ScenarioLikeTested

Reply:
Yeah, my solution was wrong, I actually needed to fix the doc-block annotation but in fact it is even better to just remove it